<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QXShapingWidget</class>
 <widget class="QWidget" name="QXShapingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QXShapingTextCombobox" name="textComboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="featuresButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="QXApplication.qrc">
       <normaloff>:/images/shape_d.png</normaloff>:/images/shape_d.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="menuButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Shaping Options...</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="QXApplication.qrc">
       <normaloff>:/images/truetype_d.png</normaloff>:/images/truetype_d.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>530</width>
        <height>465</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QXShapingGlyphView" name="glyphView" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QXShapingGlyphView</class>
   <extends>QWidget</extends>
   <header>QXShapingWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QXShapingTextCombobox</class>
   <extends>QComboBox</extends>
   <header>QXShapingTextCombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QXApplication.qrc"/>
 </resources>
 <connections/>
</ui>
